### PR TITLE
fix typo texts/unit/ops/transformer/interface

### PR DESCRIPTION
**Pull request by digger-yu**
fix typo texts/unit/ops/transformer/interface
change espilon to epsilon
detail info
        modified:   tests/unit/ops/transformer/inference/test_layer_norm.py
        modified:   tests/unit/ops/transformer/inference/test_rms_norm.py

### CODE REVIEW

File: tests/unit/ops/transformer/inference/test_layer_norm.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def test_layer_norm_residual(batch, seq_len, channels, dtype):
     assert allclose(new_output, ref_output)
 
 
-def residual_store_ref_implementation(vals, bias, res, gamma, beta, espilon, channels, dtype):
+def residual_store_ref_implementation(vals, bias, res, gamma, beta, epsilon, channels, dtype):
```
Comment (mrwyattii) [inline tests/unit/ops/transformer/inference/test_layer_norm.py:89]: `epsilon` isn't used in this function, could you instead remove it from the args and the function call below? Thanks!

Comment (digger-yu) [reply tests/unit/ops/transformer/inference/test_layer_norm.py]: Modif done ,please check again

**Comment by mrwyattii** [comment]
Closing in favor of #3697. @digger-yu added you as co-author on that one. Thank you for the contribution!

**Comment by mrwyattii** [comment]
Closing in favor of https://github.com/microsoft/DeepSpeed/pull/3697. @digger-yu added you as co-author on that one. Thank you for the contribution!

**Pull request closed without merging**